apisix/plugins/ext-plugin/helper.lua

```diff
@@ -56,4 +56,27 @@ function _M.get_conf_token_cache_time()
 end
 
 
+function _M.response_reader(reader, callback, ...)
+    if not reader then
+        return "get response reader failed"
+    end
+
+    repeat
+        local chunk, read_err, cb_err
+        -- TODO: HEAD or 304
```

Comment:
Could you use at least one sentence for this TODO? I can't recall why this comment is added.

Reply:
done